### PR TITLE
(fix) Khmer translations added and few textual changes

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary
This PR adds the translations for new texts added in #1451 .

## Screenshots
None

## Related Issue
None

### CODE REVIEW

File: packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx [hunk 1]
```diff
@@ -222,7 +222,7 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
     if (minVisitStopDatetime && visitStopDatetime <= minVisitStopDatetime) {
       validSubmission = false;
       setError('visitStopDate', {
-        message: t('invalidVisitStopDate', 'Stop date needs to be on or after {{lastEncounterDatetime}}', {
+        message: t('invalidVisitStopDateAccEncounter', 'Stop date needs to be on or after {{lastEncounterDatetime}}', {
```
Comment (ibacher) [inline packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx:225]: I don't understand the `AccEncounter` part here; I get we need to disambiguate `invalidVisitStopDate`, but `visitStopDateMustBeAfterMostRecentEncounter` seems better. Translation keys are translator-facing and should be named in such a way that they facilitate understanding the use of this key.

File: packages/esm-patient-chart-app/src/visit/hooks/useDeleteVisit.hook.tsx [hunk 2]
```diff
@@ -51,10 +51,12 @@ export function useDeleteVisit(patientUuid: string, visit: Visit, onVisitDelete
             : t('visitCancelled', 'Visit cancelled'),
           kind: 'success',
           subtitle: !isCurrentVisitDeleted
-            ? t('visitCancelSuccessMessage', 'Active {{visit}} cancelled successfully', {
+            ? t('visitDeletedSuccessfully', '{{visit}} is deleted successfully', {
```
Comment (ibacher) [inline packages/esm-patient-chart-app/src/visit/hooks/useDeleteVisit.hook.tsx:54]:
```suggestion
            ? t('visitDeletedSuccessfully', '{{visit}} was deleted successfully', {
```

**Review by ibacher — COMMENTED** [review]
@vasharma05 Is there a reason this is a draft?

**Comment by vasharma05** [comment]
> @vasharma05 Is there a reason this is a draft?

Hi @ibacher !
Actually, there were some missing translations and hence I kept the PR in draft.

All the translations have been added, please review them again.
Thanks for your time!

File: packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx (hunk 1 above)
Comment (vasharma05) [reply packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx]: Just bad choice of words on my side.
Thanks for the improvement. I'll take care of the same from the next time.

File: packages/esm-patient-chart-app/src/visit/hooks/useDeleteVisit.hook.tsx [hunk 3]
```diff
@@ -51,10 +51,12 @@ export function useDeleteVisit(patientUuid: string, visit: Visit, onVisitDelete
             : t('visitCancelled', 'Visit cancelled'),
           kind: 'success',
           subtitle: !isCurrentVisitDeleted
-            ? t('visitCancelSuccessMessage', 'Active {{visit}} cancelled successfully', {
+            ? t('visitDeletedSuccessfully', '{{visit}} was deleted successfully', {
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/visit/hooks/useDeleteVisit.hook.tsx:54]:
```suggestion
            ? t('visitDeletedSuccessfully', '{{visit}} deleted successfully', {
```

From the [design](https://zeroheight.com/23a080e38/p/683580-notifications), it looks like the verb (`was` in this case) is superfluous.

Comment (vasharma05) [reply packages/esm-patient-chart-app/src/visit/hooks/useDeleteVisit.hook.tsx]: Done!

**Pull request merged**